tests/transactional/transactional_update.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Summary: Test transactional updates
-#   Installs & remove ptf, update, rollback
+#   Installs & remove pkg, update, rollback
```

Comment:
PTF is a specific kind of package. Please, don't change the meaning of the tests.

Reply:
That was my concern, thanks for confirming this. @jknphy as we have talked, we can start with covering simple installation with zypper, rather more complex cases like program temporary fixes or installation of rpm files from the disk and not from the repo.

Reply:
changed back to ptf